.github/workflows/build.yml

```diff
@@ -27,51 +27,31 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: ["8.3"]
-                symfony: ["^5.4.21", "^6.4"]
-                sylius: ["~1.13.0", "~1.14.0"]
-                mysql: ["8.4"]
+                php: ["8.2", "8.3"]
+                symfony: ["^6.4", "^7.1"]
+                sylius: ["2.0.0"]
```

Comment:
Is it necessary to specify patch version as I think build will be stuck at 2.0.0 even after 2.0.x release?